scripts/components/Main.scss

```diff
@@ -1,197 +1,192 @@
-@import "../styles/icons.scss";
+@use "../styles/icons" as *;
```

Comment:
I'd still use aliases instead of the typical relative paths setup that will need to be changed when one changes the file structure, but that's a matter of taste. No issue.